### PR TITLE
[ #3 ] Release 1.5.0

### PR DESCRIPTION
**Pull request by joebochill**
#### Changes proposed in this Pull Request:
- Updates to jest configuration for RN projects to allow tests to pass
- Updates to NPX commands to allow them to run in npx 7 without requesting user input

~~Waiting for #70 ~~
~~Waiting for https://github.com/pxblue/react-native-cli-templates/pull/8 ( and the master version https://github.com/pxblue/react-native-cli-templates/pull/9)~~

I published a beta so we can check this with the recently published react native templates:

```
npx -p @pxblue/cli@beta pxb new react-native
```

### CODE REVIEW

**Comment by joebochill** [comment]
Having a slight issue it seems:
image

File: src/constants/dependencies.ts [hunk 1]
```diff
@@ -3,12 +3,11 @@ export const DEPENDENCIES = {
         '@angular/cdk@^10.2.5',
         '@angular/flex-layout@^10.0.0-beta.32',
         '@angular/material@^10.2.5',
-        '@pxblue/angular-themes@^5.2.0',
+        '@pxblue/angular-themes@^6.0.0-beta.0',
```
Comment (daileytj) [inline src/constants/dependencies.ts:6]: We need to update this to 6.0.1

Comment (daileytj) [reply src/constants/dependencies.ts:6]: https://github.com/pxblue/pxblue-cli/pull/72

**Comment by jeffvg** [comment]
did we get everything in for angular auth template?

image

**Comment by joebochill** [comment]
It looks like there may be an underlying issue with npm v7. I logged a bug on their repo (https://github.com/npm/cli/issues/3007) to see if we can get some insight into what might be going on. Until then, I think we will just need to have a note that says we only work with npm v6. I'll update the readme.

**Comment by joebochill** [comment]
An updated angular auth template will be published by https://github.com/pxblue/angular-cli-templates/pull/16

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 ### Fixed
 
--   Enable commands to work with npm/npx version 7 by prefixing npx commands with `npm_config_yes=true` (https://omrilotan.medium.com/npx-breaking-on-ci-b9f3f61d4676)
```
Comment (huayunh) [inline CHANGELOG.md]: Would be helpful if this is included in readme too

**Review by emclaug2 — APPROVED** [review]
Angular Projects via all templates work.

**Comment by joebochill** [comment]
RN looks good on mac with NPM 6 and NPM 7

**Review by daileytj — APPROVED** [review]

**Pull request merged**